stringutils.go

```diff
@@ -70,3 +73,33 @@ func hashsum(s string, hasher hash.Hash) string {
 	hasher.Write([]byte(s))
 	return hex.EncodeToString(hasher.Sum(nil))
 }
+
+func foundInHIBP(s string) error {
```

Comment:
Ok, this is starting to look real good! One final wish I have: I don't think this method really belongs to `stringutils.go`. I think we can either just move it to `crunchy.go` or put it into a separate `hibp.go`. I'd probably prefer the latter, as it's really hazzle-free in Go.

Reply:
alright, on it now